network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/notification/NotificationApiTest.kt

```diff
@@ -92,6 +92,20 @@ class NotificationApiTest : ApiTest {
         assertIs<EventContentDTO.Unknown>(firstEvent)
     }
 
+    @Test
+    fun givenAValidRequest_whenRegisteredValidToken_theTokenRegisteredSuccessfully() = runTest {
+        val httpClient = mockAuthenticatedHttpClient(
+            NotificationEventsResponseJson.registerTokenResponse, statusCode = HttpStatusCode.Created, assertion = {
+                assertPost()
+                assertBodyContent(VALID_PUSH_TOKEN_REQUEST.rawJson)
+            }
+        )
+
+        val notificationsApi = NotificationApiImpl(httpClient, TEST_BACKEND_CONFIG)
+
+        notificationsApi.registerToken(VALID_PUSH_TOKEN_REQUEST.serializableData)
```

Comment:
You can assert the response 
```suggestion
        val actual = notificationsApi.registerToken(VALID_PUSH_TOKEN_REQUEST.serializableData)
        assertIs<NetworkeResponse.Success>(actual)
        assertEqual({the expected response}, actual)
```